experimental/sdk/webpubsub-socketio-extension/src/SIO/components/web-pubsub-adapter.ts

```diff
@@ -72,8 +121,26 @@ export class WebPubSubAdapterInternal extends NativeInMemoryAdapter {
    * @param rooms - a set of rooms
    */
   public async addAll(id: SocketId, rooms: Set<Room>): Promise<void> {
-    // TODO: Implement this method.
-    // For now, it is intentionally empty.
+    // TODO: RT should support a new API AddConnectionsToGroups
+    debug(`addAll SocketId = ${id}, |rooms| = ${rooms.size}`);
+    try {
+      const eioSid = this._getEioSid(id);
+
+      // TODO: Temporary mitigation. This behaviour should be taken only one time and by service
+      rooms = rooms.add(""); // add namespace default room
```

Comment:
is it possible rooms already contains ""?

Reply:
Yes, user is able to  use `""` as room name. It's a problem. 
We could ban such behaviour in service side. After a connection is connected, RT will join the connection to the group whose room is `""` directly. L129~L130 could be removed. RT will do the same job.